entrypoint.sh

```diff
@@ -140,7 +108,7 @@ echo "Running Mumble server as uid=$(id -u) gid=$(id -g)"
 echo "\"${DATA_DIR}\" has the following permissions set:"
 echo "  $( stat ${DATA_DIR} --printf='%A, owner: \"%U\" (UID: %u), group: \"%G\" (GID: %g)' )"
 
-echo "Command run to start the service : ${server_invocation[@]}"
+echo "Command run to start the service : ${server_invocation[*]}"
```

Comment:
What's the difference between `[@]` and `[*]` in this case?

Reply:
For the `echo` command itself it achieves the same behavior, but a linter like ShellCheck will claim this is an error unless we explicitly ignore SC2145. The reason for it is `[@]` and `[*]` behave differently when in the middle of a string:

```bash
declare -a someArray=(1 2 3 4)

# This is how [@] is expanded when in the middle of a string (these command are the same)
echo "my array: ${someArray[@]} yay"
echo "my array: 1" "2" "3" "4 yay"

# This is how [*] is expanded when in the middle of a string (these commands are the same)
echo "my array: ${someArray[*]} yay"
echo "my array: 1 2 3 4 yay"
```

Because `[@]` generates more arguments while `[*]` doesn't, using the former in the middle of a string could have unwanted consequences.

More info here: [SC2145](https://github.com/koalaman/shellcheck/wiki/SC2145)